integrations-and-supported-tools/prophet/notebooks/Neptune_prophet.ipynb

```diff
@@ -0,0 +1,343 @@
+    "\n",
+    "* If you are running the notebook on your local machine, you need to have [Python](https://www.python.org/downloads/) and [pip](https://pypi.org/project/pip/) installed.\n",
+    "* If you want to see the example recorded to your own workspace instead:\n",
+    "    * Create a Neptune account → [Take me to registration](https://neptune.ai/register)\n",
+    "    * Create a Neptune project that you will use for tracking metadata → [Tell me more about projects](https://docs.neptune.ai/administration/projects)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "557bc3b5",
+   "metadata": {},
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f1e24601",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip install neptune-client neptune-prophet prophet==1.1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7f776cad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import neptune.new as neptune"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "813878e1",
+   "metadata": {},
+   "source": [
+    "## Initialize Neptune and create new run\n",
+    "Connect your script to Neptune application and create new run.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8f521acb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run = neptune.init(\n",
+    "    project='common/fbprophet-integration', \n",
+    "    api_token=\"ANONYMOUS\",\n",
+    "    tags=[\"fbprophet\", \"artifacts\", \"notebook\"], # optional  \n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cf32479",
+   "metadata": {},
+   "source": [
+    "Click on the link above to open this run in Neptune. For now it is empty but keep the tab with run open to see what happens next.\n",
+    "\n",
+    "You tell Neptune: \n",
+    "\n",
+    "* **who you are**: your Neptune API token `api_token` \n",
+    "* **where you want to send your data**: your Neptune `project`.\n",
+    "\n",
+    "At this point you have new run in Neptune. For now on you will use `run` to log metadata to it.\n",
+    "\n",
+    "---\n",
+    "\n",
+    "**Note**\n",
+    "\n",
+    "\n",
+    "Instead of logging data to the public project `'common/sklearn-integration'` as an anonymous user 'neptuner' you can log it to your own project.\n",
+    "\n",
+    "To do that:\n",
+    "\n",
+    "1. Get your [Neptune API token](https://docs.neptune.ai/getting-started/installation#get-api-token)\n",
+    "2. Pass the token to ``api_token`` argument of ``neptune.init()`` method: ``api_token=YOUR_API_TOKEN``\n",
+    "3. Pass your project to the ``project`` argument of ``neptune.init()``.\n",
+    "\n",
+    "For example:\n",
+    "\n",
+    "```python\n",
+    "neptune.init(project=\"YOUR_WORKSPACE/YOUR_PROJECT\",\n",
+    "             api_token=\"YOUR_API_TOKEN\")\n",
```

Comment:
We will remove passing user's own API token from examples at some point, but for now we can keep it